frontend/src/pages/modelCatalog/components/ModelCatalogCard.tsx

```diff
@@ -29,17 +29,10 @@ export const ModelCatalogCard: React.FC<{ model: CatalogModel; source: string }>
   <Card isFullHeight data-testid="model-catalog-card">
     <CardHeader>
       <CardTitle>
-        <Flex alignItems={{ default: 'alignItemsCenter' }}>
-          {model.logo ? (
-            <img src={model.logo} alt="model logo" style={{ height: '36px', width: '36px' }} />
-          ) : (
-            <Skeleton
-              shape="square"
-              width="36px"
-              height="36px"
-              screenreaderText="Brand image loading"
-            />
-          )}
+        <Flex>
```

Comment:
i made the changes here because brandimage wasn't aligning correctly due to it's wrappers and how that interacts with the flex layout. this was part of haley's touchup changes. i'll double check but i'm pretty sure this will be a regression on that.